docker/dockerfile-joex

```diff
@@ -0,0 +1,45 @@
+FROM docspell-dev:base
+
+ENV UNO_URL https://raw.githubusercontent.com/unoconv/unoconv/0.9.0/unoconv
+
+RUN apk add --no-cache openjdk11-jre \
```

Comment:
This looks similar to `joex-base.dockerfile` – is it possible to use this here? Just to avoid duplication for maintenance reasons….